test/test_rosbag/bag_migration_tests/scripts/generate_data

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Function that builds a message generation:
+build_generation()
+{
+    catkin build --no-deps --no-notify --no-status --no-summary --cmake-args -DMSG_DIRECTORY="msg_$1" -- test_rosbag \
+        >/dev/null
```

Comment:
I could try, but remember this is only to generate the data. In other words, it's an utility script that the developer would run to provide/update the tests, but it's not a dependency of the package strictly speaking.

I'd try doing that though, but where should I run `cmake` from? I'd like to keep thinks tidy and clean, with the `build` directory in the `build/PACKAGE` folder under the workspace.